modules/ovms_ai_extension/Dockerfile.lgpl

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2021 Intel Corporation
+              
+FROM openvino/model_server:2021.4.1-gpu
+USER root 
+
+RUN apt-get update && \
+    dpkg --get-selections | grep -v deinstall | awk '{print $1}' > base_packages.txt  && \
+    apt-get install -y --no-install-recommends ${DEPS} && \
+    rm -rf /var/lib/apt/lists/*
+    
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
```

Comment:
Since you removed `build-essential` from one Dockerfile, maybe you could also remove it here?

By the way, with removing `build-essential` from the image, wouldn't `third-party-programs.txt` with licenses get smaller?

Reply:
thanks, also removed here - we're keeping the third party mega-license in as is because I've been told we need to reflect packages in the base dockerfile - at least what our open source approval wanted us to do.